image_size:
  height: 480
  width: 640
camera_matrix:
  - 359.0416607440124
  - 0
  - 326.7664029189718
  - 0
  - 359.4554209546856
  - 227.4554222575176
  - 0
  - 0
  - 1
fish_eye: true
dist_coeffs:
  - -0.06170247224470661
  - 0.031007158721413
  - -0.01848443192406562
  - -0.002848753682136129
# battery
homography:
  - 0.0003894013235704503 
  - 0.07627973097664416 
  - -38.14415891341121
  - -0.07507768767967746 
  - -0.003909525546509963 
  - 25.56898829064741
  - 3.457316405613152e-06
  - -0.004648915178930289 
  - 0.9999999999999999

rvec:
  - 1.26799795343115
  - 1.299488181508184
  - -1.159490694430253

tvec:
  - -0.3779211057144259
  - 7.444061216311891
  - 29.90910655207841

delta: 0.05

# no battery
# homography:
#   - -0.0004745822392424082
#   - 0.08024380326489193
#   - -39.77140219293138
#   - -0.07643065727205575
#   - 0.0003849109666223868
#   - 25.00296059848414
#   - 9.207849146508149e-05
#   - -0.004707476122632882
#   - 1